src/main/java/com/faforever/client/chat/PircBotXChatService.java

```diff
@@ -610,6 +609,10 @@ public void close() {
     }
   }
 
+  public void close(){
+    //requested by interface
+  }
```

Comment:
Checking where this method is used reveals that it is used in unit tests. Now I don't know if it only exists for unit testing or if it was used in the past, however, it shouldn't be implemented empty here but rather removed (from the interface, too) and the unit test changed to use `destroy()`

Reply:
@DarkHorseV1 it just make unit test call destroy instead of close and remove the Method from the ChatService interface

Reply:
I will need some time